### ISSUE TITLE
Some questions

### ISSUE THREAD

**Issue by remydev** [anykeyh/clear#100]
Hello,

* I would like to know why the class is transformed in the plural when ask the the database. 
in this [exemple](https://clear.gitbook.io/project/model/associations/belongs_to)
Category is transform to categories in the query.
I find that this is subject to errors, what do you think ?

* Also, in the same example, why there isn't  "include Clear::Model" in the Category class ?

* Can I change all "with_serial_pkey" to "primary_key" in this documentation ?

Thanks

**Comment by anykeyh** [comment]
Hi Remy ! 

To answer to your points:

1. By default, the convention would be to have the table as plural of the name of the model. This convention came from Ruby on Rails' active record framework that I used before I made this gem. I thought it's a good convention and I kept with it in Clear. If you're not happy, you might want to rename your table manually by adding this code in your models:

```crystal
class XXX
  include Clear::Model
  
  self.table = "xxx"
end
```

2. Probably an error of mine When I wrote the documentation code, I eventually didn't test everything I wrote - shame on me! -.

3. According to point 2 and you diving down into documentation, I would say: I would be glad if you contribute to make the documentation or the shard better  😄 . Feel free to contribute, at your pace, and you will have all my respect 👍 . 
Currently I'm super busy with my work (I might have time in few months), so any contributions is more than welcomed !
